components/cam/src/physics/cam/micro_p3_utils.F90

```diff
@@ -85,8 +85,8 @@ module micro_p3_utils
 real(rtype), parameter :: dsph = 3._rtype
 
 ! Bounds for mean diameter for different constituents.
-real(rtype), parameter :: lam_bnd_rain(2) = 1._rtype/[500.e-6_rtype, 20.e-6_rtype]
-real(rtype), parameter :: lam_bnd_snow(2) = 1._rtype/[2000.e-6_rtype, 10.e-6_rtype]
+! real(rtype), parameter :: lam_bnd_rain(2) = 1._rtype/[500.e-6_rtype, 20.e-6_rtype]
```

Comment:
Can we just remove these lines?

Reply:
I'm ok with that. I just left them there assuming upcoming p3 stuff would need them, so people would only have to uncomment them. @AaronDonahue , thoughts?

Reply:
I'm OK keeping them if you think they are useful.

Reply:
I don't know if they are. Hopefully Aaron can shed some light.